lib/vector/Vlib/buffer2.c

```diff
@@ -1144,13 +1146,13 @@ void Vect_point_buffer2(double px, double py, double da, double db,
     double angular_tol, angular_step, phi1;
     int j, nsegments;
 
-    G_debug(2, "Vect_point_buffer()");
+    G_debug(2, "%s()", __func__);
 
     *oPoints = Vect_new_line_struct();
 
     dalpha *= PI / 180; /* convert dalpha from degrees to radians */
 
-    if (round || (!round)) {
+    if (round) {
```

Comment:
Round is a bool or an int? If it is an int, what's the Boolean value when it's not 0 or 1?

Reply:
Boolean value as int is either 0 or 1.

Reply:
But in practice, if you throw any int as a boolean value, only `0` is false, everything else is true.

Reply:
Ok, so no other choice than truthy or falsy (like a nan in doubles), so the code before didn't do anything.

Reply:
> Ok, so no other choice than truthy or falsy (like a nan in doubles), so the code before didn't do anything.

Yes, the condition was always true. Every time this function was called the true if-statement was executed. But, the function was/is only called if `round` was true.